frontend/src/features/properties/selector/search/PropertySelectorSearchContainer.test.tsx

```diff
@@ -39,87 +44,149 @@ describe('PropertySelectorSearchContainer component', () => {
 
     return {
       store,
-      component,
+      ...utils,
     };
   };
 
+  beforeEach(() => {
+    mockAxios
+      .onGet(
+        new RegExp(
+          'https://openmaps.gov.bc.ca/geo/pub/WHSE_CADASTRE.PMBC_PARCEL_FABRIC_POLY_SVW/wfs',
+        ),
+      )
+      .reply(200, mockPropertyLayerSearchResponse)
+      .onGet(
+        new RegExp(
+          'https://openmaps.gov.bc.ca/geo/pub/WHSE_ADMIN_BOUNDARIES.TADM_MOT_REGIONAL_BNDRY_POLY/wfs',
+        ),
+      )
+      .reply(200, mockMotiRegionLayerResponse)
+      .onGet(
+        new RegExp(
+          'https://openmaps.gov.bc.ca/geo/pub/WHSE_ADMIN_BOUNDARIES.TADM_MOT_DISTRICT_BNDRY_POLY/wfs',
+        ),
+      )
+      .reply(200, mockDistrictLayerResponse)
+      .onGet(new RegExp('tools/geocoder/parcels/pids/*'))
+      .reply(200, mockGeocoderPidsResponse)
+      .onGet(new RegExp('tools/geocoder/addresses'))
+      .reply(200, mockGeocoderOptions);
+  });
+
```

Comment:
This mocks requests to various services consumed by the form being tests. A bit of an improvement over a generic `mockAxios.onGet().reply(...)` that applies to every web request